packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -1408,3 +1423,78 @@ test.serial('time jumps forward', async t => {
   t.is(schedules.liveAuctionSchedule?.startTime.absValue, 1530n);
   t.is(schedules.liveAuctionSchedule?.endTime.absValue, 1550n);
 });
+
+// serial because dynamicConfig is shared across tests
+// This test reproduced bug #8296. Now its expectations don't match the behavior
```

Comment:
we need a regression test. so please change this test to ensure that a missing price feed does not throw.